utility/reimage_octo_nodes.py

```diff
@@ -0,0 +1,302 @@
+"""
+Usage:
+  reimage_octo_nodes.py [--username=<username>] [--password=<password>] --os_version=<version> --nodes=<nodes>
+  reimage_octo_nodes.py (-h | --help)
+
+Options:
+  -h --help          Show this help message and exit
+  --username=<username>   Username for authentication
+  --password=<password>   Password for authentication
+
+Examples:
+  # Install OS version 8.6 on nodes cali0xx1 and cali0xx2 with custom username and password
+  reimage_octo_nodes.py --version=8.6 --nodes=cali0xx1 cali0xx2 --username=myusername --password=mypassword
+"""
+
+import multiprocessing
+import subprocess
+import time
+
+import paramiko
+import requests
+from docopt import docopt
+
+# Disable the InsecureRequestWarning
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+from ceph.ceph import CommandFailed
+from utility.retry import retry
+
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+# Define the base URL and credentials
+base_url = "https://{}"
+username = "xxxx"
+password = "xxxx"
+
+# Define a dictionary to store OS versions and their respective ISO URLs
+image_list = {
+    "9.2": "http://10.0.209.99/rhel92ks_updated.iso",
```

Comment:
Can we use magna006 - which is reserved for Baremetal Infra

Reply:
Installed httpd server on magna006 and updated the iso there.